types/bootstrap/js/dist/alert.d.ts

```diff
@@ -0,0 +1,38 @@
+declare class Alert {
+    constructor(element: Element);
+
+    /**
+     * Closes an alert by removing it from the DOM. If the .fade and .show
+     * classes are present on the element, the alert will fade out before it
+     * is removed.
+     */
+    close: () => void;
```

Comment:
Are these supposed to be function-valued fields or methods? Making them function-valued fields affects subclassability and also affects assignability.

Reply:
These properties are methods.

Reply:
Then I would suggest writing them as this:
```suggestion
    close(): void;
```

Reply:
Thank you, I updated PR